src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const inverseRob = {};
+
+  for (const key in robot) {
+    if (inverseRob[robot[key]]) {
+      return null;
+    } else {
+      inverseRob[robot[key]] = key;
+    }
```

Comment:
```suggestion
    } 
    inverseRob[robot[key]] = key;
```

Reply:
you don't need else here :)

if your condition will work - it'll just return null and program stop
if skip - it will do what you want without else :)